src/rollup/types.d.ts

```diff
@@ -91,12 +91,14 @@ export interface PluginCache {
 }
 
 export interface PluginContext {
+	// deprecate:
 	watcher: Watcher;
 	addWatchFile: (id: string) => void;
 	cache: PluginCache;
 	resolveId: ResolveIdHook;
 	isExternal: IsExternal;
 	parse: (input: string, options: any) => ESTree.Program;
+	// deprecate asset dependencies
```

Comment:
I didn't even know that assets could have dependencies and it's already being deprecated 🤔😞